chroma_core/migrations/0020_clientmounts_remove_fk.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+def resolve_fk(apps, schema_editor):
+    ClientMount = apps.get_model("chroma_core", "LustreClientMount")
+    ManagedFilesystem = apps.get_model("chroma_core", "ManagedFilesystem")
+    for m in ClientMount.objects.all():
+        m.filesystem = ManagedFilesystem.objects.get(id=m.filesystem).name
+        m.save()
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("chroma_core", "0019_auto_20200529_2026"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="lustreclientmount",
+            name="filesystem",
+            field=models.CharField(help_text=b"Mounted filesystem", max_length=8),
+        ),
```

Comment:
Just to confirm, the `AlterField` / `RemoveField` sections were generated by `./manage.py makemigrations`?

Reply:
Yes. Just checked it from scratch.